modules/internal/ChapelLocale.chpl

```diff
@@ -611,8 +756,9 @@ module ChapelLocale {
   // Its type should probably be renamed dummyLocale or something
   // representative.
   // The dummy locale provides system-default tasking and memory management.
-  pragma "no doc"
-  const dummyLocale = new unmanaged DummyLocale();
+  /*pragma "no doc"*/
+  /*const dummyLocale = new unmanaged DummyLocale();*/
```

Comment:
If `dummyLocale` isn't used any more we should just remove it.  (It seems likely to be unused since there are 2 decls both of which are commented out.)